docs.phtml

```diff
@@ -698,23 +698,5 @@ $older7		="3";
 </ul>
 </div>
 
-<div class="featurecontainer">
```

Comment:
Guides were last updated respectively 2001 and 2002, and they can be reinstated by popular demand, but I think the biggest supporters will already have secured their copy by now.

Reply:
How about replacing the content by a link to https://wiki.gnucash.org/wiki/Documentation?

Reply:
Frank, the section contains two user guides from version 1.6. There is NO reason to include them on the website or the wiki. The content is both outdated and superceded.

Reply:
There is a sidebar entry for the wiki, and "Documentation" is in both "Downloads" and "Support".
I think Download should be a button, and "Documentation" relegated to the "Support" section only.